src/Frontend/Components/AttributionColumn/ButtonRow.tsx

```diff
@@ -50,35 +57,45 @@ export function ButtonRow(props: ButtonRowProps): ReactElement {
   const classes = useStyles();
 
   return (
-    <div className={classes.buttonRow}>
-      {props.showButtonGroup ? (
-        <ButtonGroup
-          isHidden={props.areButtonsHidden}
-          mainButtonConfigs={getMainButtonConfigs(
-            props.temporaryPackageInfo,
-            props.isSavingDisabled,
-            props.onSaveButtonClick,
-            props.onSaveForAllButtonClick,
-            Boolean(props.showSaveForAllButton)
-          )}
-          contextMenuButtonConfigs={getContextMenuButtonConfigs(
-            props.packageInfoWereModified,
-            props.onDeleteButtonClick,
-            props.onDeleteForAllButtonClick,
-            Boolean(props.hideDeleteButtons),
-            props.onUndoButtonClick,
-            Boolean(props.showSaveForAllButton)
-          )}
-        />
-      ) : (
-        <ToggleButton
-          buttonText={'hide'}
-          className={classes.resolveButton}
-          selected={props.selectedPackageIsResolved}
-          handleChange={props.resolvedToggleHandler}
-          ariaLabel={'resolve attribution'}
-        />
-      )}
-    </div>
+    <>
```

Comment:
```suggestion
    <div>
```

This fixes the left-alignment (see previous comment) with the content above on Mac.